### PR TITLE
chore: update all scripts for docker compose v2

### PR DESCRIPTION
**Pull request by bodymindarts**
Due to high demand for compatibility with docker compose v2 I think its time we upgraded our scripts.

### CODE REVIEW

File: DEV.md [hunk 1]
```diff
@@ -14,8 +14,8 @@ $ jq --version
 jq-1.6
 $ docker --version
 Docker version 20.10.7, build f0df350
-$ docker-compose --version
-docker-compose version 1.29.2, build unknown
+$ docker compose version
+Docker Compose version 2.0.0, build unknown
```
Comment (nicolasburtey) [inline DEV.md:18]:
```suggestion
Docker Compose version 2.0.0
```

**Review by nicolasburtey — DISMISSED** [review]
works for me

**Comment by bodymindarts** [comment]
Had to get compose v2 installed in ci/image - hence force push @nicolasburtey

File: DEV.md (hunk 1 above)
Comment (bodymindarts) [reply DEV.md]: done

**Review by nicolasburtey — APPROVED** [review]

**Review by adam-stover — APPROVED** [review]
works

**Review by vindard — APPROVED** [review]
Works on Linux as well

**Pull request merged**